src/Wizard.js

```diff
@@ -120,6 +122,7 @@ const Wizard = ({ children, onComplete, stateManager, debug }: Props) => {
           updateStep,
           stateManager,
         }}>
+        <Step name="empty"> </Step>
```

Comment:
This ensures that the Wizard does not render the first step of the client's Wizard prior to restoring state.

Reply:
@rix1 and I talked and I realised that this commit would break isFirst() and isLast() functionality. Need to investigate this.

Reply:
b0a001528c89bd36e49537e675ecdcc75ce92763 another attempt at a solution for this fix.

Reply:
There were two obvious solutions for this problem:
- create a "loading" step in Losen which is displayed on load - this means that client code does not run until the stateManager kicks in and loads the correct step. However, this meant added some "magic" code confusion and additional logic would be required in isFirst(). This also means that a "fake" step.

- initialise index with null and set the correct step onLoad. This meant that we had to do some defensive checks to appease Flow. This option also means that when loading from a statemanager, the active step must be provided or the step will never load (as opposed to rendering the first step by default). This must be handled by the client but this is an acceptable restriction.

The first solution of having a "fake step" requires less code changes, but increases "mystery".
The second solution has less mystery, but decreases readability slightly.

The above commit fixes this issue with the second solution.